### PR TITLE
Use confirmed/prompted flags with setConsent

### PR DESCRIPTION
**Pull request by eligrey**
This resolves potential sync issues with DoNotSellDisclosure when combined with an upcoming airgap.js version

## Related Issues

- Links https://transcend.height.app/T-21602
- Links https://transcend.height.app/T-21509

## Security Implications

_[none]_

## System Availability

_[none]_

### CODE REVIEW

**Review by michaelfarrell76 — DISMISSED** [review]
cool, i know  we had this convo on slack, but it makes me think that customers with a custom UI may experience this issue still

**Comment by eligrey** [comment]
@michaelfarrell76 Fortunately I don't think anyone is using custom UI with the latest isPrompted() call included. Also, there aren't any customers using a custom UI + IAB USP just yet. If you're not using IAB USP then you don't need the latest UI changes supporting isPrompted sync

**Comment by michaelfarrell76** [comment]
oh didnt realize this is iab specific

File: index.html [hunk 1]
```diff
@@ -65,7 +65,11 @@ <h2>View States</h2>
         transcend.doNotSell(event);
       };
       var resetDoNotSell = (event) => {
-        airgap.setConsent(event, { SaleOfInfo: true });
+        airgap.setConsent(
+          event,
+          { SaleOfInfo: true },
+          { confirmed: true, prompted: true },
```
Comment (anotherminh) [inline index.html:71]: the function name is `resetdoNotSell` - does it make sense that we are setting confirmed + prompted to true if we're trying to reset the state

**Review by anotherminh — DISMISSED** [review]

File: index.html (hunk 1 above)
Comment (eligrey) [reply index.html:71]: good catch. I just removed the consent options for this call 👍

Comment (eligrey) [reply index.html:71]: note that this file is just stubs btw

**Review by anotherminh — DISMISSED** [review]

**Review by anotherminh — APPROVED** [review]

**Pull request merged**